src/samples/annotation-list/v1/first-page.json

```diff
@@ -50,6 +50,33 @@
                     ]
                 }
             ]
+        },
+        {
+            "id": "Rpp6JttuEeeXnRsvD_86eQ",
+            "access": "public",
+            "document": {
+                "title": "Molecular basis for multimerization in the activation of the epidermal growth factor",
+                "uri": "https://elifesciences.org/articles/14107"
+            },
+            "parents": [],
+            "created": "2017-09-15T15:40:52Z",
+            "highlight": "clustering are important correlates of EGFR activation"
+        },
+        {
+            "id": "g8IV4ttXEeewdPeQMRBLqQ",
+            "access": "restricted",
+            "document": {
+                "title": "Molecular basis for multimerization in the activation of the epidermal growth factor",
+                "uri": "https://elifesciences.org/articles/14107"
+            },
+            "parents": [],
+            "created": "2017-09-15T16:40:00Z",
+            "content": [
+                {
+                    "type": "paragraph",
+                    "text": "Paragraph text"
+                }
+            ]
```

Comment:
It would be great if we could have failing examples in this project but locally I did add an item with not `highlight` or `content` and it threw an error when I tried to compile it.

Reply:
Yeah, always meant to introduce a larger test suite with negative tests.